snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java

```diff
@@ -65,6 +67,13 @@ public void createRegularVersion() {
 		createVersion(INT_CODESYSTEM, "regular-version", getNextAvailableEffectiveDate(INT_CODESYSTEM)).statusCode(201);
 		assertGetVersion(INT_CODESYSTEM, "regular-version").statusCode(200);
 	}
+	
+	@Test
+	public void createRegularVersionWithAuthor() {
+		Map<String, String> headers = Map.of(AbstractRestService.X_AUTHOR, "info@b2international.com");
+		createVersion(INT_CODESYSTEM, "version-with-author", getNextAvailableEffectiveDate(INT_CODESYSTEM), headers).statusCode(201);
+		assertGetVersion(INT_CODESYSTEM, "version-with-author").statusCode(200);
```

Comment:
Could you please verify here that the commit author is properly registered? Thanks!